src/Site_Command.php

```diff
@@ -651,6 +651,82 @@ public function unspam( $args ) {
 		$this->update_site_status( $args, 'spam', 0 );
 	}
 
+	/**
+	 * Set one or more sites to mature.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <id>...
+	 * : One or more IDs of sites to set as mature.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     $ wp site mature 123
+	 *     Success: Site 123 set to mature.
+	 *
+	 * @subcommand mature
```

Comment:
This `@subcommand mature` declaration isn't necessary when it mirrors the name of the method. `@subcommand` is only used when the command name differs from the method name.